$NetBSD: patch-bj,v 1.6 2005/12/15 21:28:48 wennmach Exp $

Cast "size" to (unsigned long long) to avoid a compiler warning on amd64.

--- tools/release-tools/tar-rootify.c.orig	2005-12-14 11:21:11.000000000 +0100
+++ tools/release-tools/tar-rootify.c	2005-12-14 11:22:34.000000000 +0100
@@ -118,7 +118,7 @@
 	    fprintf(stderr, "gname = %.*s\n", (int)sizeof(p->gname), p->gname);
 	    fprintf(stderr, "type =     %c\n", p->typeflag);
 	    fprintf(stderr, "size = %.*s\n", (int)sizeof(p->size), p->size);
-	    fprintf(stderr, "size = %llo\n", size);
+	    fprintf(stderr, "size = %llo\n", (unsigned long long)size);
 	}
 
 	hcksum = estrntoll(p->chksum, 8, 8);
